tests/scripts/all.sh

```diff
@@ -886,6 +888,63 @@ clang_version() {
     fi
 }
 
+gcc_version() {
+    gcc="$1"
+    if command -v "$gcc" > /dev/null ; then
+        "$gcc" --version | sed -En '1s/^[^ ]* \([^)]*\) ([0-9]+).*/\1/p'
+    else
+        echo 0  # report version 0 for "no gcc"
+    fi
+}
+
+can_run_cc_output() {
```

Comment:
this is a very strange name for this function - it's not "output" from "can run cc", or even "can we run" "cc output" - how should I be interpreting this?

Reply:
in fact, just `can_run_cc` would see to fit what this function does - is that right?

Reply:
It's checking whether we can execute the binary produced by the C compiler, passed as the first argument. Hence "cc output". 

But I'm open to changing the name, if you prefer something else

Reply:
Oh, I see. So "can we run" "compiler output (executable)". Heh, this is like one of those pictures that can either be two faces or a vase - I can see it either way, but when I see it in one way the other doesn't make sense. However, I don't have a better suggestion - we're checking if the compiler produces output for the current platform, I guess